DigitalOffice/Services/ProjectService/Startup.cs

```diff
@@ -4,14 +4,14 @@
 using Microsoft.Extensions.Configuration;
```

Comment:
You lost your interfaces configuration methods after merge with develop.

Reply:
fixed